config/dev.exs

```diff
@@ -34,6 +34,11 @@ config :recognizer, RecognizerWeb.Endpoint,
     ]
   ]
 
+config :recognizer,
+  hal: [
```

Comment:
I'm not sure we need to worry about nesting this config. `config, :recognizer, hal_url: ""` should suffice. That'll also make retrieving it a tad easier.